src/hooks/internal/useForm.js

```diff
@@ -1,11 +1,17 @@
 import React, { useContext } from "react";
+import { ERROR_STRINGS } from "../../utils/error";
 
 const FormContext = React.createContext();
 
 export const FormProvider = FormContext.Provider;
 
 function useForm() {
-  return useContext(FormContext);
+  const context = useContext(FormContext);
+  if (context) {
+    return context;
+  } else {
```

Comment:
Unnecessary `else`

Reply:
The `useContext(FormContext)` does return null if the `Input` or whatever field component is used without a `Form` component. Or would your prefer it written in a different way?

Reply:
Definitely worth checking if there is context. All I'm saying is that you could omit the `else`:
```
if (context) {
  return context;
}

throw new Error(...);
```
or
```
if (!context) {
  throw new Error(...);
}

return context;
```

Reply:
oh right my bad, I misinterpreted that. Yeah that makes sense 👍